uniffi_bindgen/src/bindings/kotlin/gen_kotlin/custom.rs

```diff
@@ -100,6 +64,10 @@ pub struct KotlinCustomType {
 impl KotlinCustomType {
     pub fn new(name: String, builtin: Type, config: Option<CustomTypeConfig>) -> Self {
         Self {
+            outer: Type::Custom {
+                name: name.clone(),
+                builtin: builtin.clone().into(),
+            },
```

Comment:
Curious (to have it in this PR).

Could this be passed in from the oracle?

Reply:
It could be but it's kind of awkward because of how `iter_custom_types()` is defined.  I really feel like there's one last refactor to do that would clean up a bunch of code, but that can be done after we merge this.